test/specs/reference/tests/11.delete.js

```diff
@@ -4,14 +4,18 @@ exports.execute = function (options) {
         var catalogId = process.env.DEFAULT_CATALOG,
             schemaName = "delete_schema",
             tableName = "delete_table",
-            reference;
+            assocTableName = "association_table";
 
         var baseUri = options.url + "/catalog/" + catalogId + "/entity/" + schemaName + ':' + tableName;
 
-        describe('when deleting regularly', function() {
+        var associationUri = options.url + "/catalog/" + catalogId + "/entity/" + schemaName + ':' + assocTableName;
+
+        describe('when deleting one row', function() {
+            var reference, tuples;
+
             beforeAll(function (done) {
                 var uri = baseUri + "/key_col=1";
```

Comment:
Just to make sure we're actually creating the url and not using the reference url, we could just change this test case to not have the filter. Also make sure that the table has more than one row, then delete one row and see if the other rows exist or not. All the test cases are currently deleting all the rows that are available on the table (which is not testing the actual change in this PR).